package.json

```
@@ -136,25 +158,37 @@
   "scripts": {
     "lint": "npm-run-all -p -c lint-js lint-css",
     "lint-js": "eslint .",
+    "lint-ts": "eslint ./js/renovation/*.{ts,tsx} ./js/renovation/**/*.{ts,tsx} ./testing/jest/**/*.{ts,tsx}",
     "lint-css": "stylelint styles",
-    "lint-staged": "lint-staged",
+    "lint-staged": "lint-staged && npm run lint-ts",
```

Move this to `.lintstagedrc` file.
```suggestion
    "lint-staged": "lint-staged",
```